core/src/foundations/ionic.vars.scss

```
@@ -939,6 +367,578 @@ $ion-primitives-pink-1100: var(--ion-primitives-pink-1100, #8b2248);
 $ion-primitives-pink-1100-rgb: var(--ion-primitives-pink-1100-rgb, 139, 34, 72);
 $ion-primitives-pink-1200: var(--ion-primitives-pink-1200, #621833);
 $ion-primitives-pink-1200-rgb: var(--ion-primitives-pink-1200-rgb, 98, 24, 51);
+$ion-semantics-primary-100: var(--ion-semantics-primary-100, $ion-primitives-blue-100);
+$ion-semantics-primary-100-rgb: var(--ion-semantics-primary-100-rgb, 242, 244, 253);
```

```suggestion
$ion-semantics-primary-100-rgb: var(--ion-semantics-primary-100-rgb, $ion-primitives-blue-100-rgb);
```

Should this fallback to `$ion-primitives-blue-100-rgb` (and the same for all other semantic rgb vars)?